cli_reference/developer_cli_odo/understanding-odo.adoc

```diff
@@ -1,5 +1,5 @@
 [id="understanding-odo"]
-= Understanding `{odo-title}`
+= Understanding odo
```

Comment:
Our other tool names are wrapped in back ticks (for example, `opm`, `tkn`), so I recommend the same here: `odo`

Reply:
Thanks, fixed!